exercises/perfect-numbers/src/test/scala/PerfectNumbersTest.scala

```diff
@@ -0,0 +1,47 @@
+import org.scalatest.{FlatSpec, Matchers}
+
+class PerfectNumbersTest extends FlatSpec with Matchers {
```

Comment:
It would be nice if there was a canonical-data.json for the exercise. But, there are already 14 languages that already implement this exercise. I don't think we should wait on approval of the canonical data. Though, we could create the file and push up a pull request. And, then update our test suite when there is an "approved" version.